test-harness/createMasterFiles.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const requests = require('./requests');
+
+const { makeRequest, constructMasterFileName } = require('./helpers');
+
+async function recordMasterFile({ path, method, headers, body }) {
+  const reqRes = await makeRequest({ path, method, headers, body });
+
+  try {
+    fs.writeFileSync(
+      `${__dirname}/gold-master-files/${constructMasterFileName({
+        path,
+        method,
+        headers,
+        body,
+      })}.json`,
+      JSON.stringify(reqRes, null, 2)
```

Comment:
Can you make sure to put a new line after the stringified content, so GitHub stops to complain?